ports/esp32/modesp32.c

```diff
@@ -228,6 +260,9 @@ static const mp_rom_map_elem_t esp32_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_mcu_temperature), MP_ROM_PTR(&esp32_mcu_temperature_obj) },
     #endif
     { MP_ROM_QSTR(MP_QSTR_idf_heap_info), MP_ROM_PTR(&esp32_idf_heap_info_obj) },
+    #if CONFIG_FREERTOS_USE_TRACE_FACILITY
+    { MP_ROM_QSTR(MP_QSTR_idf_task_stats), MP_ROM_PTR(&esp32_idf_task_stats_obj) },
```

Comment:
Seeing this new function right next to `idf_heap_info()`, maybe the new one should be called `idf_task_info()` for consistency?

After all, it is documented as "Returns information about ..." :smile: 

No strong feelings here, up to you.